root yarn install fails on node 16

The `re2` dep fails to build under node 16, looks like a fix landed recently https://github.com/uhop/node-re2/issues/100

```console
~/Code/ipfs-shipyard/nft.storage on main  
❯ yarn 
...snip...
In file included from /Users/oli/Library/Caches/node-gyp/16.1.0/include/node/v8.h:30:
/Users/oli/Library/Caches/node-gyp/16.1.0/include/node/v8-internal.h:452:38: error: no template named 'remove_cv_t' in namespace 'std'; did you mean 'remove_cv'?
            !std::is_same<Data, std::remove_cv_t<T>>::value>::Perform(data);
                                ~~~~~^~~~~~~~~~~
                                     remove_cv
/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/include/c++/v1/type_traits:776:50: note: 'remove_cv' declared here
template <class _Tp> struct _LIBCPP_TEMPLATE_VIS remove_cv
                                                 ^
1 error generated.
make: *** [Release/obj.target/re2/lib/addon.o] Error 1
gyp ERR! build error 
```

npm does not include deprecated versions in it's version resolution. yarn, it seems does.

you can only recreate this problem in the `get-urls` module if you use yarn.
```console
~/Code/olizilla/get-urls on main  
❯ npm i 
~/Code/olizilla/get-urls on main  
❯ npm ls re2                                                                                                                                       11:57:52
get-urls@10.0.0 /Users/oli/Code/olizilla/get-urls
└─┬ url-regex-safe@1.0.2
  └── re2@1.16.0
~/Code/olizilla/get-urls on main  
❯ rm -rf node_modules 
~/Code/olizilla/get-urls on main  
❯ yarn                                                                                                                                             14:19:24
yarn install v1.22.10
info No lockfile found.
[1/5] 🔍  Validating package.json...
[2/5] 🔍  Resolving packages...
warning url-regex-safe@1.1.0: this package has been deprecated
...
In file included from /Users/oli/Library/Caches/node-gyp/16.1.0/include/node/v8.h:30:
/Users/oli/Library/Caches/node-gyp/16.1.0/include/node/v8-internal.h:452:38: error: no template named 'remove_cv_t' in namespace 'std'; did you mean 'remove_cv'?
            !std::is_same<Data, std::remove_cv_t<T>>::value>::Perform(data);
                                ~~~~~^~~~~~~~~~~
                                     remove_cv
/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/include/c++/v1/type_traits:776:50: note: 'remove_cv' declared here
template <class _Tp> struct _LIBCPP_TEMPLATE_VIS remove_cv
                                                 ^
1 error generated.
make: *** [Release/obj.target/re2/lib/addon.o] Error 1
gyp ERR! build error 
```

This is my recurring experience of using yarn. Some nice logging and some significant and suprising differences in module resolution from npm. _grumble_

--- a/package.json
+++ b/package.json
@@ -34,5 +34,8 @@
     "rimraf": "^3.0.2",
     "simple-git-hooks": "^2.3.1",
     "typescript": "^4.2.4"
+  },
+  "resolutions": {
+    "url-regex-safe": "^2.0.2"
   }
 }

